packages/html-elements/src/elements/Footer.tsx

```diff
@@ -0,0 +1 @@
+export { default as Footer } from '../primitives/View';
```

Comment:
It looks like `contentinfo` would be the appropriate `accessibilityRole` to apply to this element for web. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role

Reply:
Ah nvm, it looks like you're creating an actual footer element in the file below this. If you were to take the same approach that you did with Article, then the comment above would be relevant.